tarantool_test.go

```diff
@@ -4001,21 +4022,52 @@ func TestConnect_schema_update(t *testing.T) {
 	defer conn.Close()
 
 	for i := 0; i < 100; i++ {
-		fut := conn.Do(NewCallRequest("create_spaces"))
+		func() {
+			fut := conn.Do(NewCallRequest("create_spaces"))
```

Comment:
Concurrent `create_spaces` may conflict with each other.

The idea is that `create_spaces` is conturrent to a next connect. We don't need goroutines here because `conn.Do()` is async.

Reply:
There is no goroutines created. I used `func()` just so I could use `defer` inside a for loop without too much repeating (or I could move `Connect` call, cancel a context and then go to if-else branches). But there is no goroutines.

Reply:
It is confusing.

I think you could just call `defer cancel()` in the loop. And it will be called x100 at the end of the tests - nothing will change.